### PR TITLE
Bug 1162349 - add integration tests for context menu of tv

### PR DESCRIPTION
**Pull request by huchengtw-moz**

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=516e388bd41a00b342917c98404e4d9de3288538

File: tv_apps/smart-system/test/marionette/browser_context_menu_test.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
+/* global __dirname */
```
Comment (shamenchens) [inline tv_apps/smart-system/test/marionette/browser_context_menu_test.js]: nit: with `node: true` in `.jshintrc`, we could remove this line.

File: tv_apps/smart-system/test/marionette/browser_context_menu_test.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+var APP_URL = 'app://' + APP_HOST;
+
+var Keys = {
+  'enter': '\ue006',
+  'right': '\ue014',
+  'esc': '\ue00c'
+};
+
+var assert = require('chai').assert;
+var SmartSystem = require(__dirname + '/lib/system');
```
Comment (shamenchens) [inline tv_apps/smart-system/test/marionette/browser_context_menu_test.js]: Should we add comment on why we don't use `client.loader.getAppClass('system');`?

Comment (huchengtw-moz) [inline tv_apps/smart-system/test/marionette/browser_context_menu_test.js]: Sure, it's important here.

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=90beb7412fb8833fb318bc8cc5e21e8d5d8323cb

**Pull request merged**